### PR TITLE
dx-87 added more info on termsofserviceUrl

### PR DESCRIPTION
**Pull request by marinafrydenb**

### CODE REVIEW

File: _includes/payment-order-purchase.md [hunk 1]
```diff
@@ -108,15 +108,15 @@ Content-Type: application/json
...
-|  ✔︎︎︎︎︎  | └─➔&nbsp;`termsOfServiceUrl`      | `string`     | The URI to the merchant terms of service document the payer must accept in order to complete the payment. This is not the Swedbank Pay Terms of Service. Requires `https`.                                                                                                                                                                                                                                    |
+|  ✔︎︎︎︎︎  | └─➔&nbsp;`termsOfServiceUrl`      | `string`     | The URI to the merchant terms of service document the payer must accept in order to complete the payment. Note that this is not required unless `generateReferenceToken` or `generateRecurrenceToken` is also submitted in the request. This is the Merchants, not the Swedbank Pay Terms of Service. Requires `https`.                                                                                                                                                                                                                                    |
```
Comment (asbjornu) [inline _includes/payment-order-purchase.md:119]:
```suggestion
|  ✔︎︎︎︎︎  | └─➔&nbsp;`termsOfServiceUrl`      | `string`     | The URI to the merchant's terms of service document, which the payer must accept in order to complete the payment. Note that this is not required unless `generateReferenceToken` or `generateRecurrenceToken` is also submitted in the request. This is the merchant's, not the Swedbank Pay's terms of service. Requires `https`.                                                                                                                                                                                                                                    |
```

File: checkout/other-features.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ underlying payments or transaction.
...
-|  ✔︎︎︎︎︎  | `termsOfServiceUrl` | `string` | The URI to the terms of service document the payer must accept in order to complete the payment. **HTTPS is a requirement**.                                                                                                                                                                             |
+|  ✔︎︎︎︎︎  | `termsOfServiceUrl` | `string` | The URI to the terms of service document the payer must accept in order to complete the payment. **HTTPS is a requirement**. Note that this is not required unless `generateReferenceToken` or `generateRecurrenceToken` is also submitted in the request. This is the Merchants, not the Swedbank Pay Terms of Service.                                                                                                                                                                             |
```
Comment (asbjornu) [inline checkout/other-features.md:54]: This description does not match the one above. Do they need to be different, or should we perhaps add a `field-description-termsofserviceurl.md` include for `termsOfServiceUrl` so they become equal?

File: checkout/other-features.md [hunk 3]
```diff
@@ -112,7 +112,7 @@ Content-Type: application/json
...
-| └➔&nbsp;`termsOfServiceUrl` | `string` | A URI that contains your terms and conditions for the payment, to be linked on the payment page.                                                                                                                                                                                                         |
+| └➔&nbsp;`termsOfServiceUrl` | `string` | A URI that contains your terms and conditions for the payment, to be linked on the payment page. Note that this is not required unless `generateReferenceToken` or `generateRecurrenceToken` is also submitted in the request. This is the Merchants, not the Swedbank Pay Terms of Service.                                                                                                                                                                                                         |
```
Comment (asbjornu) [inline checkout/other-features.md:115]: Not equal to the previous descriptions. Intentional or not?

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5285058&bid=17302606)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5285058&bid=17302606)

File: _includes/field-description-termsofserviceurl.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    unless `generateReferenceToken` or `generateRecurrenceToken` is also submitted
+    in the request. This is the Merchants, not the Swedbank Pay Terms of
+    Service. **HTTPS is a requirement**.
+{%- endcapture -%}
+{{- termsofserviceurl | strip_newlines -}}
+
+{%- comment -%}
+The dashes in the Liquid code tags remove output whitespace. More on that here:
+
+https://shopify.github.io/liquid/basics/whitespace/
```
Comment (payex-dev) [inline _includes/field-description-termsofserviceurl.md:13]: Codacy Issue found: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5285058)

**Pull request merged**